core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2724,6 +2750,7 @@ public String toString() {
           .setDefaultValue(Constants.S3_MULTIPART_TEMPORARY_DIR_SUFFIX)
           .setDescription("Suffix for the directory which holds parts during a multipart upload.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.NONE)
```

Comment:
why is this `Scope.NONE`? Shouldn't this be `Scope.SERVER`?

Reply:
Previously all the Proxy properties are given `Scope.NONE`,  I actually not quite sure what scope should be given to Proxy properties. `Scope.NONE` and `Scope.SERVER` both make sense